### PR TITLE
Add function call param alignment span and thresh

### PR DESCRIPTION
**Pull request by SuhaibI**
Added span and threshold options to supplement the "align_same_func_call_params" setting. This provides better configurability for function call alignment, and is in line with existing alignment options.

### CODE REVIEW

File: src/options.h [hunk 1]
```diff
@@ -2524,6 +2524,18 @@ align_func_params_gap;
...
 
+// The span for aligning function-call parameters for single line functions.
+//
+// 0 = Don't align (default)
+extern BoundedOption<unsigned, 0, 5000>
+align_same_func_call_params_span;
+
+// The threshold for aligning function-call parameters for single line
+// functions.
+// 0 = No limit (default)
```
Comment (mwoehlke-kitware) [inline src/options.h]: Not: should probably be a "blank" line here?

```
// The threshold for aligning function-call parameters for single line
// functions.
//
// 0 = No limit (default)
```

Comment (mwoehlke-kitware) [reply src/options.h]: Actually, another nit: we aren't quite consistent with whether or not there should be a period at the end of e.g. `0 = No limit (default)`, but we use one more often than not, so please add that also. (I'm going to make another PR shortly to clean up the inconsistency.)

**Comment by mwoehlke-kitware** [comment]
Thanks for the doc tweaks!

**Comment by mwoehlke-kitware** [comment]
It looks like the tests are now failing. Did you update the expected test output by hand, or did you run `make update-cli-options` / `ninja update-cli-options`?

**Comment by SuhaibI** [comment]
> It looks like the tests are now failing. Did you update the expected test output by hand, or did you run `make update-cli-options` / `ninja update-cli-options`?

Derp, did it manually. Sorry about that, I should've re-tested before committing. Fixed.

**Pull request merged**